### ISSUE TITLE
China CNY / BTC  Market Maker Bounty 1000 $BSQ

### ISSUE THREAD

**Issue by flix1** [bisq-network/growth#135]
### China CNY / BTC Market Maker Bounty

To claim this bounty the contributor must make a market in Bisq CNY/BTC for 1 month.
https://bisq.network/markets/?currency=btc_cny

This will involve:

1. Keeping at least **2 BUY** and **2 SELL** offers live during 1 month.

2. Attracting other traders to the market through marketing, communication, social media...

3. Completing at least 5 trades.

4. Report with feedback here on the details of the trades (payment method used, problems encountered, suggestions for improvement, fees paid, time it took to complete the trade).

5. Attend at least 1 live Bisq Growth call to answer questions and provide feedback on the experience.

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/58692026-fecd3700-838d-11e9-8be7-4d0cbde44401.png)

It looks like https://bisq.network/ is once again blocked by the Great Firewall of China. I guess that you can still work around it with a VPN... but better to put this idea on hold for the time being.

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/61549810-42205980-aa51-11e9-81ac-9ce42af7055a.png)
We've had some activity in China over the past few days. 

The legal situation of Bitcoin in China has also had positive developments recently:
https://twitter.com/DoveyWan/status/1151944343999008768

Apparently a property dispute case in a Chinese court has resulted in a verdict which declares Bitcoin akin to a commodity and therefore legally protected property.

**Comment by jmaurice** [comment]
Since when does Bisq care about the "legal standing" of Bitcoin in some country? IMO the entire purpose of Bisq is to not have to care what local government thinks/says/does about Bitcoin. The relevant issue at hand is that Tor, GitHub, and Bisq's domain name are all blocked in China, so Bisq is essentially censored there. IMO we should implement Tor bridges or other necessary infrastructure to circumvent China's censorship of Bisq before worrying about liquidity problems or offering any bounties for this.

**Comment by flix1** [comment]
> IMO the entire purpose of Bisq is to not have to care what local government thinks/says/does about Bitcoin.

Agreed.

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/63943425-ef828600-ca6f-11e9-8c66-e7d1d630eb0b.png)

Some offers still live... but no trades.

**Comment by flix1** [comment]
@wdljt6 @ff98sha do you know any traders in China who might be interested in this bounty?

**Comment by ff98sha** [comment]
> @wdljt6 @ff98sha do you know any traders in China who might be interested in this bounty?

After discussion with @wdljt6 , we think that we may be unavailable to take this bounty in the near future, as for various problems we face.

Let me just put forward some background information about BTC trade in China, hoping that will benefit your further work.
# BTC trade is illegal now in China. But a week ago the government announced that blockchain is to be of great importance during the development of China.
# The Bisq website is currently not blocked by Great Firewall, while Tor network could only be connected with VPN or meek bridge.
# Huobi (huobi.vn) and Binance (binance.co) are two of the biggest BTC trade platform used by Chinese. You may find more information there. VPN is needed to visit these two websites in China.
# In China, there have already been some cases that defines selling/using VPN as an illegal behaviour, but that's just a small porpotion compared to the majority VPN users.

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/68579993-b4d89900-0475-11ea-9481-556678f993ec.png)
https://twitter.com/DoveyWan/status/1192972887935746048

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/69237179-5dcd7500-0b95-11ea-8e0d-6ee407589278.png)

There's now 2 different buyers and 3 offers in CNY. Still no sellers... but I get the feeling this market is very close to starting up.

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/69421177-a49ba600-0d20-11ea-83d1-38652db59e71.png)

And we finally have a spread in China!

**Comment by wdljt6** [comment]
If the biaq needs,I can keep 2 or 3 seller and buyer.

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/71886958-67a73980-313d-11ea-829d-a27a81d8b7a4.png)

1st trade of the year in CNY!

**Comment by flix1** [comment]
Something is moving in China!

![image](https://user-images.githubusercontent.com/4233784/72249073-ed712c00-35f8-11ea-85f1-48f267a69a03.png)


![image](https://user-images.githubusercontent.com/4233784/72249109-fb26b180-35f8-11ea-8210-478396aaa4e6.png)

**Comment by RadishXN** [comment]
> ### China CNY / BTC Market Maker Bounty
> To claim this bounty the contributor must make a market in Bisq CNY/BTC for 1 month.
> https://bisq.network/markets/?currency=btc_cny
> 
> This will involve:
> 
> 1. Keeping at least **2 BUY** and **2 SELL** offers live during 1 month.
> 2. Attracting other traders to the market through marketing, communication, social media...
> 3. Completing at least 5 trades.
> 4. Report with feedback here on the details of the trades (payment method used, problems encountered, suggestions for improvement, fees paid, time it took to complete the trade).
> 5. Attend at least 1 live Bisq Growth call to answer questions and provide feedback on the experience.
> 
> ### Bounty: 1000 $BSQ

Hi,I would like to take part in this issue. So how can I star with? And who should I talk with? Thx

**Comment by flix1** [comment]
Hi @RadishXN just writing here saying that you intend to claim the bounty is enough to start with.

The 5 conditions are clear. Just start being a market maker.

**Comment by RadishXN** [comment]
> Hi @RadishXN just writing here saying that you intend to claim the bounty is enough to start with.
> 
> The 5 conditions are clear. Just start being a market maker. If you want examples of previously claimed similar bounties take a look at #138 and #160
> 
> After you have been a market maker for 1 month and have fulfilled the 5 conditions, write here detailing your activity and if everyone is satisfied you will be instructed to create a compensation request and claim the bounty.
> 
> At worst, if the result is unsatisfactory, you will be asked to do some more. But it should not be so difficult to keep 4 orders live for 1 month and complete 5 trades.

Thanks again, I have already started my work. I would like to report as soon as the first deal happening.

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/72425834-cfd2cc80-3788-11ea-9b61-4191b661ac5a.png)

![image](https://user-images.githubusercontent.com/4233784/72425851-d95c3480-3788-11ea-819f-a4e3967fa870.png)

Nice to see those offers live @RadishXN

**Comment by RadishXN** [comment]
> ![image](https://user-images.githubusercontent.com/4233784/72425834-cfd2cc80-3788-11ea-9b61-4191b661ac5a.png)
> 
> ![image](https://user-images.githubusercontent.com/4233784/72425929-06104c00-3789-11ea-8c6b-cea10ed622c8.png)
> 
> Nice to see those offers live @RadishXN
Thanks, now I am waiting for someone to purchase them.

**Comment by flix1** [comment]
@RadishXN remember that it helps to **communicate**... if you don't get the word out about Bisq, nobody will know that your offers are there. 

A good place to start might be in Chinese Bitcoin OTC chats... you don't need much. Just a couple of buyers/sellers over the next month will get you there. A good example to follow would be what was done with the JPY bounty. #138

**Comment by RadishXN** [comment]
> @RadishXN remember that it helps to **communicate**... if you don't get the word out about Bisq, nobody will know that your offers are there.
> 
> A good place to start might be in Chinese Bitcoin OTC chats... you don't need much. Just a couple of buyers/sellers over the next month will get you there. A good example to follow would be what was done with the JPY bounty. #138
> 
> Also feel free to ask for help. if something does not work or could be changed to help you... now is the time to try it.

Thanks for your asking. In last few days,it's Chinese new year. And after that, it will be a great time to advertise bsq. So I have already found some way to advertise it. So I will try my best to do it. I believe there will be some purchase soon. And thanks again for your asking.

**Comment by flix1** [comment]
No offers on BTC/CNY since Feb 2 2020. 
https://bisq.network/markets/?currency=btc_cny

This coincides with new Chinese financial controls and market restrictions. There is some discussion on keybase about how to bypass restrictions.

I suppose it's only fair to consider this bounty paused until the problem is solved.

I consider this market more important than ever as a test of Bisq's censorship resistance. If people can trade BTC on Bisq despite the GFW... then Bisq really is antifragile.

**Comment by mys67** [comment]
Just so you guys know, the Chinese love red. Their stock market shows gains as red, not green.
I would recommend inverting your color scheme, when the language is set to Chinese.
Also, there is a large Monero mining community there. If you had xmr as a base pair you would probably have more volume, as those guys want to convert their XMR back into CNY after mining. But, they may want to convert CNY if their country continues to pump money into their market.
Another insight, there are a lot of Chinese in the United States with access to CNY and obviously access to tor without configuring a tor relay. Just some food for thought.

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/74135100-fa7b3e00-4beb-11ea-81fa-a78cac224de5.png)

Offers are back on CNY/BTC!

Apparently the bounty hunters found a workaround to bypass the blockade.

Bounty is back on!

**Comment by RadishXN** [comment]
> ![image](https://user-images.githubusercontent.com/4233784/74135100-fa7b3e00-4beb-11ea-81fa-a78cac224de5.png)
> 
> Offers are back on CNY/BTC!
> 
> Apparently the bounty hunters found a workaround to bypass the blockade.
> 
> Bounty is back on!

Nice to hear that, now I carry on the Bounty. And try to get more people to bsq.

**Comment by flix1** [comment]
Feb 13th 2020. 

![image](https://user-images.githubusercontent.com/4233784/74436363-99ab6a00-4e66-11ea-8c72-a4283bbf075e.png)

**Comment by RadishXN** [comment]
I claim this bounty.
> 1. Keeping at least **2 BUY** and **2 SELL** offers live during 1 month.

I have keep 2 BUY and 2 SELL alive. And so far I still keep 2 BUY and 2 SELL alive.

> 2. Attracting other traders to the market through marketing, communication, social media...

1. I have done some introduction about what is bisq. And do some research about Bisq and Chinese BTC market.Then I found that China have a wide market that need we to active it, which means maybe more AD will be a choice. Some people need this kind of market to hide thier ID but they can't found this right APP.

2. I have get some users from telegram. And Chinese would like to do some purchase  avioding providing their ture ID on telegram.It's a huge market but so few of them heard about Bisq making this app can't take it advantage.

3. Alilpay and wechat pay take a huge market of Chinese pay,but that doesn't mean Bisq don't have chance. Instead some Chinese are realize the importance of hiding their ID. So I believe Chinese market will grow prosperous soon. 

> 3. Completing at least 5 trades.

As the picture said, I have done 6 offers,and because my introduce,some chinese for the first time know the Bisq and would like to take part in it.
![20200213234538](https://user-images.githubusercontent.com/45912686/74455209-f25b2100-4ebf-11ea-9a0e-16edc05e4670.png)

> 4. Report with feedback here on the details of the trades (payment method used, problems encountered, suggestions for improvement, fees paid, time it took to complete the trade).

1. The network of Bisq make a immense obstacle of Chinese users. And even to me it is a litter annoying. And as I have said that wechat pay and Ailipay has a very convenient usage experience for the most of Chinese.So the network of Bisq will be the most significant part to prove. I mean Chinese do have a large market .If Bisq want take part of it,then the improve of network is absolutly needed. Ban by GFW,which means lose a huge amount of users.

2. The Bisq is based on Java. It will take a sea amount of memory of computer. It may need to improve .But more important is that most Chinese don't have enough expericence to run a sever that can put their offer alive all day.And believe me the price of Chinese local severs are ridiculously high. So it's hard for quite a lot of Chinese to put thier offers alive. I don't know if there is a way to improve it, but I do believe it need improve.

3. The lackness of adverting. Adverting is needed,China have a great market. So Bisq is a great APP,but so few of Chinese know it.Because we don't get enough advertisement. Maybe not some 
AD that cost a huge pirce but at least a AD that can make people need Bisq or hide their ID know it.
I believe this is important. ps. maybe on some telegram channel.


> 5. Attend at least 1 live Bisq Growth call to answer questions and provide feedback on the experience.

I will participant the next Growth call.But I don't know the time of it.

Thank you.

**Comment by flix1** [comment]
Thank you very much for all the work and for your great feedback @RadishXN ! 
We know how difficult it is to start a new market from zero and how complicated it is to explain how it all works to completely new users. 

Please don't forget to create a compensation request for cycle 11.

@wiz @m52go do you have more questions for the bounty hunter? or any ideas on how he can continue to grow this market?

**Comment by RadishXN** [comment]
> Thank you very much for all the work and for your great feedback @RadishXN !
> We know how difficult it is to start a new market from zero and how complicated it is to explain how it all works to completely new users.
> 
> Please don't forget to create a compensation request for cycle 11.
> 
> @wiz @m52go do you have more questions for the bounty hunter? or any ideas on how he can continue to grow this market?

Yes, I have already put that in cycle 11 at here https://github.com/bisq-network/compensation/issues/494
And thanks again for the help.If there is anything I can help with in the furture I will glad to help.

**Comment by m52go** [comment]
Great work! Nice to see some action here, and thanks for the detailed feedback. @flix1 thanks for pushing for this.

@RadishXN is there a way to help with marketing? If we do another bounty, I would prefer to combine it with a stronger marketing effort this time to build on the momentum you've started. If we don't do anything differently, I imagine we can't expect different (i.e., even better) results.

You mentioned Telegram...do you think it would be helpful to make some infographics to share there that illustrate the benefits of Bisq to help people understand visually? Maybe make some guides (videos?) to help people run Bisq in China (i.e., how did you get around restrictions?). 

Also, is English viable for the target audience, or would this communication need to be in Chinese?

**Comment by RadishXN** [comment]
> Great work! Nice to see some action here, and thanks for the detailed feedback. @flix1 thanks for pushing for this.
> 
> @RadishXN is there a way to help with marketing? If we do another bounty, I would prefer to combine it with a stronger marketing effort this time to build on the momentum you've started. If we don't do anything differently, I imagine we can't expect different (i.e., even better) results.
> 
> So perhaps next bounty includes a little more for marketing efforts. Maybe infographics to share on Telegram, guides (videos?), etc. Need to discuss more. I sent you a message on Keybase.

Yes,some channel on telegram will offer Chinese some service like buy something needing to hide their ID. Such as sim card, IT information, even something that might not legal. So that's the reason they use it. But some people may only want buy a way to break the obstacle of Internet. And without hide ID something like this will also be banned.So that maybe the next market.

**Comment by m52go** [comment]
Update: @RadishXN, the Chinese translators, and I discussed an approach to broaden Bisq usage in China yesterday. 

- the workaround that enables Bisq to be beat the firewall seems to work well but needs a little guidance/education to help others get it working
- RadishXN will explore groups with biggers audiences likely to use Bisq and arrange for Bisq to be promoted there
- RadishXN and translators will work on videos to explain Bisq and how to use it in China, which we will post on the Bisq YouTube channel
  - subtitles to 2 introductory videos have already been translated
- when the topic of translating documentation came up, I mentioned the new Bisq wiki, as it allows for anyone to add and manage content at will (current asciidoctor-based docs framework makes implementing translations rather unwieldy)

Once RadishXN determines a promotion strategy that makes sense, we'll determine guidelines for another market maker bounty and shoot for more activity this time around.

**Comment by flix1** [comment]
Closing as completed successfully.
